test/e2e/test-upgrade-sqlite.sh

```
@@ -49,8 +49,7 @@ podman push "${QUAY_ENDPOINT}/init/upgrade-test:v1" --tls-verify=false
 
 # Upgrade to current build
 log_info "Upgrading to current build..."
-${MIRROR_REGISTRY_DIR}/mirror-registry upgrade -v \
-    --quayHostname "${QUAY_ENDPOINT}"
+${MIRROR_REGISTRY_DIR}/mirror-registry upgrade -v
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)